actions/setup-tfenv-terraform/action.yml

```diff
@@ -9,8 +9,12 @@ runs:
   using: composite
   steps:
     - run: |
-        git clone --depth=1 https://github.com/tfutils/tfenv.git ~/.tfenv
-        echo "PATH=$HOME/.tfenv/bin:$PATH" >> "$GITHUB_ENV"
+        if ! tfenv --version; then
+          git clone --depth=1 https://github.com/tfutils/tfenv.git ~/.tfenv
+          echo "PATH=$HOME/.tfenv/bin:$PATH" >> "$GITHUB_ENV"
+        else
+          echo "Terraform already initialized, skipping install"
+        fi
```

Comment:
Since we're doing an if/else you can drop the `!` and put the "skipping install" first. I would also reword that as "tfenv already installed" since terraform initialization will still happen in the next step.